### DISCUSSION TITLE
Script command for center a object:

### DISCUSSION THREAD

**Discussion by StellariumSentinel** [Stellarium/stellarium#3583] [Q&A] [answered]
Dear all,
Is there a way to emulate pressing the "Space" key on the keyboard from a script? I would like to center an object in the center of the screen before zooming in ... I looked in the script help, but I couldn't find the right command. Can you kindly help me?

Thank you so much everyone!

**Comment by alex-w** [comment, marked as answer]
Yes, https://stellarium.org/doc/23.0/classStelMainScriptAPI.html#a589ac48c07c4fecfd13a784d29499a8c

**Reply by StellariumSentinel** [reply]
Great Alex! Thank you so much! I solved! :-)
